package.json

```diff
@@ -65,7 +67,7 @@
     "logs:infra": "./scripts/start/logs.sh",
     "stop": "./scripts/stop.sh",
     "nuke": "./scripts/nuke.sh",
-    "lint": "yarn packages --parallel run lint",
+    "lint": "yarn packages --parallel run lint --rule '@typescript-eslint/no-require-imports: warn'",
```

Comment:
Plaese don't use specific rules here. If needed please put that in a local config file for this package.

Reply:
ok so I see and I think we should _not_ have this here but just in the eslint config files where it seems to be needed.